### PR TITLE
docs: ui capabilities toolbar PTL-1582

### PR DESCRIPTION
**Pull request by Gareth-Spencer-Genesis**
ATENTION! YOU SHOULD NOW BRANCH FROM PREPROD WHEN YOU UPDATE 

  - Please check the [internal contributions guide](https://www.notion.so/genesisglobal/Contributing-new-documentation-75953fb245f246ff872789035451a0c4) for details
  
__________

Thank you for contributing to the documentation.

Do the changes you have made apply to both Current and Previous versions?
No

Have you done a trial build to check all new or changed links?
Yes

Is there anything else you would like us to know?
No

**This week's exciting advice from the style guide**

- Write your headings in sentence case:

  - A good example
    This is a correct heading.
  - A Bad Example
    This is not a correct heading.

### CODE REVIEW

File: examples/ui/client-capabilities/interaction/toolbar.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	const isBrowser = useIsBrowser();
+
+	if (isBrowser) {
+		const RapidImports = require('../../rapidImports');
+		RapidImports.registerComponents();
+	}
+
+	return (
+		<CodeSection>
+			<rapid-toolbar>
```
Comment (patrickoneill-genesis) [inline examples/ui/client-capabilities/interaction/toolbar.js:15]: can we have an example or horizontal and vertical position, and start and end slots and with examples too

File: docs/001_develop/03_client-capabilities/008_interaction/toolbar.mdx [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+Property and attribute binding examples for Genesis Component syntax.
+
+<table>
+  <thead><tr><th>Name</th><th>Type</th><th>Description</th><th>Example</th></tr></thead>
+  <tbody>
+    <tr>
+      <td>orientation</td>
+      <td><code>ToolbarOrientation</code></td>
+      <td>The orientation of the toolbar, `horizontal` or `vertical`.</td>
```
Comment (matteematt) [inline docs/001_develop/03_client-capabilities/008_interaction/toolbar.mdx:178]: What is the default?

File: docs/001_develop/03_client-capabilities/008_interaction/toolbar.mdx [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+        <rapid-option>Option 1</rapid-option>
+        <rapid-option>Second option</rapid-option>
+        <rapid-option>Option 3</rapid-option>
+      </rapid-select>
+      <rapid-radio-group>
+        <rapid-radio checked="">One</rapid-radio>
+        <rapid-radio>Two</rapid-radio>
+        <rapid-radio>Three</rapid-radio>
+      </rapid-radio-group>
+    </rapid-toolbar>
```
Comment (matteematt) [inline docs/001_develop/03_client-capabilities/008_interaction/toolbar.mdx:44]: Same for Genesis, React, and Angular - I think we only need the tag here
```suggestion
<rapid-toolbar></rapid-toolbar>
```

**Review by matteematt — COMMENTED** [review]
Two small bits

**Review by matteematt — APPROVED** [review]

**Pull request merged**